tests/python/frontend/pytorch/test_forward.py

```diff
@@ -2893,6 +2911,7 @@ def forward(self, inp):
 @tvm.testing.uses_gpu
 def test_simple_rnn():
     """test_simple_rnn"""
+
```

Comment:
nit: I suppose that all such white spaces are redundant. Could you please remove them in this test and in the tests below?

Reply:
Sorry I didn't see it, I've reverted it now.